registry/storage/shares.go

```diff
@@ -58,7 +58,6 @@ type sharesStorage struct {
 	db             basedb.Database
 	prefix         []byte
 	shares         map[string]*types.SSVShare
-	operators      map[uint64][]byte
```

Comment:
not used anymore?

Reply:
I guess we have this data already in `ValidatorStore` and `operatorStorage`, so this is legacy.